pom.xml

```diff
@@ -118,6 +118,11 @@
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-configuration-processor</artifactId>
```

Comment:
I see what you mean now. Ok, lets leave it like this. But if I understand this Spring feature correctly, this is only to get auto-completion and hints when you use these properties?